### ISSUE TITLE
LanguageClientStart: [LC] invalid type: string "All", expected u8

### ISSUE THREAD

**Issue by vargheseg** [autozimu/LanguageClient-neovim#946]
- Did you upgrade to latest plugin version?
Yes

- Did you upgrade to/compile latest binary? Run shell command
  `bin/languageclient --version` to get its version number.
languageclient 0.1.155

- (Neovim users only) Did you check output of `:checkhealth LanguageClient`?
Yes, no errors here

- Did you check [troubleshooting]?
Yes

## Describe the bug
`:LanguageClientStart` does not work; have the following message:

![[LC] invalid type: string "All", expected u8 ](https://user-images.githubusercontent.com/19566373/71896612-266a5600-314c-11ea-8790-80ae4ad273e7.png)
- The above is seen when trying to start the client on a c++ file.
- The problem seems to be introduced in commit 13729a41570a2226f98f4eb617c0fa356fd7bf2a; could not reproduce the issue on the commit just before this one.  

## Environment
- neovim/vim version (`nvim --version` or `vim --version`):
```
NVIM v0.5.0-dev
Build type: RelWithDebInfo
Lua 5.1
Compilation: /usr/bin/x86_64-linux-gnu-gcc -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -O2 -g -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wvla -fstack-protector-strong -fdiagnostics-color=auto -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/build/neovim-JdFics/neovim-0.5.0+ubuntu1+git201912292018-34a5924-00e710e/build/config -I/build/neovim-JdFics/neovim-0.5.0+ubuntu1+git201912292018-34a5924-00e710e/src -I/build/neovim-JdFics/neovim-0.5.0+ubuntu1+git201912292018-34a5924-00e710e/.deps/usr/include -I/usr/include -I/build/neovim-JdFics/neovim-0.5.0+ubuntu1+git201912292018-34a5924-00e710e/build/src/nvim/auto -I/build/neovim-JdFics/neovim-0.5.0+ubuntu1+git201912292018-34a5924-00e710e/build/include
Compiled by buildd@lcy01-amd64-014

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info

```

- This plugin version (`git rev-parse --short HEAD`):
`d273439`
- Minimal vimrc content (A minimal vimrc is the smallest vimrc that could
  reproduce the issue. Refer to an example [here][min-vimrc.vim]):
```
  let g:LanguageClient_serverCommands = {
    \ 'c': ['/bin/ccls', '--log-file=/tmp/ccls.log'],
    \ 'cpp': ['/bin/ccls', '--log-file=/tmp/ccls.log'],
    \ }
```
- Language server link and version:

[min-vimrc.vim]: https://github.com/autozimu/LanguageClient-neovim/blob/next/min-vimrc.vim

## To Reproduce
- Run `:LanguageClientStart` on a cpp file.

You should get message:
[LC] invalid type: string "All", expected u8

**Comment by 414owen** [comment]
Getting the same error here:

```
#######
LanguageClient 0.1.155
#######
13:22:36 WARN unnamed src/language_server_protocol.rs:2284 Failed to start language server automatically. invalid type: string "All", expected u8
```

**Comment by martskins** [comment]
I think this can be attributed to the fact that there wasn't a new version release with the commit mentioned here. If you manually build (`make release` on the LanguageClient-neovim folder) it works again. Note that `./install.sh` won't work precisely because it sees that the version hasn't changed so it skips re-building.
This is probably my fault, as I wasn't sure when/how the version bump should happen, so maybe I should have included that in the PR. Apologies for the inconvenience.

**Comment by autozimu** [comment]
Just released 0.1.156. Please run `./install.sh` to fetch the latest binary.

Thanks fro reporting.
